piff/interp.py

```diff
@@ -173,7 +173,7 @@ def _finish_write(self, fits, extname):
         :param fits:        An open fitsio.FITS object
         :param extname:     The base name of the extension
         """
-        pass
+        raise NotImplementedError("Derived classes must define the _finish_write method.")
```

Comment:
iirc we had figured _finish_write might be a method that wasn't needed by all classes, and so if it wasn't needed, then it just wouldn't do anything...